tests/utils/emoticons.test.jsx

```diff
@@ -27,10 +27,5 @@ describe('Emoticons', () => {
             expect(Emoticons.handleEmoticons('asdf:goat:asdf:dash:asdf', new Map(), emojis)).
                 toEqual('asdf$MM_EMOTICON0asdf$MM_EMOTICON1asdf');
         });
```

Comment:
Should remove third argument (`emojis`) in the above tests

Reply:
Ah, nice catch